controllers/funding/programmes.js

```diff
@@ -140,21 +140,11 @@ function initProgrammesList(router, options) {
 function renderProgrammeDetail({ res, entry }) {
     addPreviewStatus(res, entry);
 
-    /**
-     * Allow for programmes without heroes
-     */
-    const defaultProgrammeHeroImage = createHeroImage({
```

Comment:
This picture no longer exists, using `heroImages.rathlinIslandDevelopment` as it doesn't have any people in it so is the safest as a fallback. We should never really show this anyway as all programmes should have their own image.